docs/website/docs/community/index.md

```diff
@@ -1,17 +1,34 @@
 # Community projects
 
+Projects built by community members:
+
+*   The [SHARK](https://github.com/nod-ai/SHARK) project from
+    [nod.ai](https://nod.ai/) uses a forked version of IREE
+    ([SHARK-Runtime](https://github.com/nod-ai/SHARK-Runtime)), offering
+    highly tuned performance on a large corpus of machine learning programs.
+
 *   The [IREE Bare-Metal Arm Sample](https://github.com/iml130/iree-bare-metal-arm)
-    demonstrates how build IREE with the
+    shows how to build IREE with the
     [Arm GNU Toolchain](https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain)
     for bare-metal Arm targets using the open-source firmware libraries
     [CMSIS](https://github.com/ARM-software/CMSIS_5) and
     [libopencm3](https://github.com/libopencm3/libopencm3).
 
 *   The [IREE C++ Template](https://github.com/iml130/iree-template-cpp)
-    demonstrates how to integrate IREE into a third-party project with CMake.
-    The project demonstrates the usage of runtime support.
+    shows one way to integrate IREE's runtime into a project with CMake.
+
+Official repositories:
+
+*   [iree-jax](https://github.com/iree-org/iree-jax) is home to
+    IREE's support for [JAX](https://github.com/google/jax) programs.
+
+*   [iree-torch](https://github.com/iree-org/iree-torch) contains
+    IREE's [PyTorch](https://pytorch.org/) frontend, leveraging the
+    [torch-mlir](https://github.com/llvm/torch-mlir) project.
```

Comment:
@dellis23 FYI, I'm adding a few links to PyTorch repos here (relates to https://github.com/iree-org/iree/pull/10630)

Reply:
Great, thank you!